trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/propeller_noise_frequency.py

```diff
@@ -0,0 +1,327 @@
+# propeller_noise_frequency.py
+# 
+# Created:  May 2016, Carlos
+# Modified: 
+
+# ----------------------------------------------------------------------        
+#   Imports
+# ----------------------------------------------------------------------    
+
+import numpy as np
+import scipy
+import matplotlib.pyplot as plt
+
+from scipy import special
+from SUAVE.Core import Units, Data
+
+
+def propeller_noise_frequency(noise_data):
+    
+    #-----------------------------------------------------
+    #unpack
+    
+    number_sections    = noise_data.number_sections                   # number of section on the blade
+    blades_number      = noise_data.blades_number                     # number of blades
+    propeller_diameter = noise_data.propeller_diameter / Units.feet   # propeller diameter [ft]
+    angle              = noise_data.angle                             # Angle from the observer
+    dist_obser         = noise_data.distance                          # Distance from an observer
+    r0                 = noise_data.r0 / Units.feet                   # Radii
+    airfoil_chord      = noise_data.airfoil_chord / Units.feet        # Chord in [ft]
+    lift_coefficient   = noise_data.lift_coefficient[:][1]
+    drag_coefficient   = noise_data.drag_coefficient[:][1]    
+    n_harmonic         = noise_data.n_harmonic                        # Number of harmonics to consider (Default = 10)
+    
+    # Unpack maxed things
+    sound_speed        = np.max(noise_data.sound_speed) / Units['ft/s']        # Feet/s
+    density            = np.max(noise_data.density)     / Units['slugs/ft**3']
+    thrust             = np.max(noise_data.thrust)      / Units.lb             #Thrust [lbs]
+    hpower             = np.max(noise_data.power)       / Units.hp             #horsepower [hp]
+    rpm                = np.max(noise_data.rpm)         * Units.rpm            #shaft rotation frequency [rad/s]
+    velocity           = np.max(noise_data.velocity)    / Units['ft/s']        #flight speed    [ft/s]
+    
+    # UNPACK MACH
+    Mx = velocity/sound_speed                   #flight Mach number
+    Mach_tip = frequency*tip_radius/sound_speed #tip rotational Mach number
+    
+    
+    MCA = 0.5*np.ones(number_sections)  #MCA (midchord of alignment or sweep) of the blade for each section
+    FA = np.zeros(number_sections)  #face alignment or offset    
+
+    p_base=2.0*1e-5  
+    zeff = 0.8
+    
+    #observer distance from propeller axis
+    dist_axis = dist_obser * np.sin(angle)                    
+    
+    
+    ######## UPDATE THIS
+    tip_radius = 0.3048*(propeller_diameter/2.0) #propeller tip radius [m]
+    
+    
+    
+    frequency = 2*np.pi*(rpm/60.)                       #Frequency of the rotation blade
+    frequency_doppler = frequency/(1-Mx*np.cos(angle))    #Doppler effect
+    
+    
+    
+    convection_factor = (1.0-Mx*np.cos(angle))  #convection factor
+    
+    #print "f = ", frequency
```

Comment:
Remove comments

Reply:
I removed this file from this PR and will put it in a noise PR